include/groonga/groonga.h

```diff
@@ -611,7 +619,12 @@ struct _grn_obj {
 
 #define GRN_OBJ_REFER    (0x01 << 0)
 #define GRN_OBJ_OUTPLACE (0x01 << 1)
-#define GRN_OBJ_OWN      (0x01 << 5)
+/**
+ * A flag to obtain ownership of the referenced `grn_obj *`. When the
+ * GRN_OBJ_OWN flag is set, the object will be automatically closed using
+ * \ref grn_obj_close when it is released.
```

Comment:
```suggestion
 * A flag to represents that \ref GRN_PTR or \ref GRN_PVECTOR owns associated \ref grn_obj.
 * When this flag is set, the associated \ref grn_obj will be automatically closed using
 * \ref grn_obj_close when owner \ref GRN_PTR or \ref GRN_PVECTOR is closed.
 *
 * You can use this flag only with \ref GRN_PTR_INIT.
```

Reply:
fix: [3a1db0f](https://github.com/groonga/groonga/pull/1944/commits/3a1db0fb2d5dce20a0b8486f4a78f30f1ed2f8d1) Thanks you for explaining it.